app/browser/tabs.js

```diff
@@ -706,6 +706,7 @@ const api = {
   },
 
   executeScriptInBackground: (script, cb) => {
+    // Do not edit without security review
```

Comment:
this is not a web-accessible api and is only used in the browser process. Actually now that we have worker threads it isn't used at all